vignettes/hazardItems.Rmd

```diff
@@ -148,10 +156,21 @@ template(templateId,layerId,attrs)
 
 ```
 
+
+#Adding Layers to the Portal 
+
+The `addLayer` function accepts a zipped shapefile and returns an itemID. Users can use this function to post new shapefiles to the Portal. The zipped shapefile must be put into your environment's `extdata` directory and the package `hazardItems` must be rebuilt to be available to R. If the user is not currently authenticated, the `checkAuth` function will be called and the user will be asked for a valid username and password
+
+To Use:
+```{r,eval=FALSE}
+addLayer(system.file("extdata", "Sandy_CIDA.zip", package="hazardItems"))
+```
+
+
 #Rendering
 ##Thumbnail Generation 
 
-The `thumb.service` accepts a json URL and creates a handsome thumbnail preview of the layer. For the CCH Portal, the thumbnails are used on the <a href="http://marine.usgs.gov/coastalchangehazardsportal/publish/item/">mediation page</a> as an item preview, in your bucket and in the search results panel.
+The `thumb.service` accepts a json URL and creates a handsome thumbnail preview of the layer. For the CCH Portal, the thumbnails are used on the <a href="https://marine.usgs.gov/coastalchangehazardsportal/publish/item/">mediation page</a> as an item preview, in your Bucket and in the search results panel.
```

Comment:
ahh ok, thanks @jread-usgs -- if it becomes an issue I will update them to use markdown!